src/icons/general/IconSearchJuicy.js.flow

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray40 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const IconSearchJuicy = ({ className = 'icon-search', color = bdlGray40, height = 14, title, width = 14 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-search ${className}`}
```

Comment:
I think `icon-search` should be `icon-search-juicy`?

Reply:
I'm honestly not sure if people use these classes but likely best to stay with the current pattern. Although I think these are older icons and they were intended to eventually be migrated to our internal Box/design-assets repo which auto-generates icons for BUIE.